renpy/display/layout.py

```diff
@@ -432,6 +432,8 @@ def __init__(self, cols, rows, padding=None,
                  transpose=False,
                  style='grid',
                  allow_underfull=None,
+                 right_to_left=False,
```

Comment:
Now that we're in py3 we can stop the accidental creep of new positional args by making any new ones kwarg-only.
```suggestion
                 *,
                 right_to_left=False,
```